### PR TITLE
Create InfinteDirectoryList scroll

### PR DESCRIPTION
**Pull request by seta-tuha**

### CODE REVIEW

**Comment by atb91590** [comment]
@seta-tuha Please add a storybook

**Comment by atungw** [comment]
This should also not be using setTimeout, don't use setState inside of a setTimeout.

File: packages/veritone-react-common/src/components/InfiniteDirectoryList/index.js [hunk 1]
```diff
@@ -0,0 +1,182 @@
...
+                    <div className={styles['table-row-text']}>
+                      {header}
+                    </div>
+                  </TableCell>
+                ))
+              }
+            </TableRow>
+          </TableHead>
+          <TableBody className={classes.tablebody}>
+            {files.map(({ id, type, name, date, selected }) => {
```
Comment (atb91590) [inline packages/veritone-react-common/src/components/InfiniteDirectoryList/index.js]: I've updated the specs to use id/type. `type` will either be 'folder' or 'tdo'. When it is 'tdo, you will have to look into `file.primaryAsset.contentType` to figure out the which icon to use.

Here is an example of how you can determine which icon to use: 
https://github.com/veritone/veritone-sdk/blob/master/packages/veritone-react-common/src/components/FilePicker/FileProgressList/index.js#L42

File: packages/veritone-react-common/src/components/InfiniteDirectoryList/story.js [hunk 2]
```diff
@@ -0,0 +1,41 @@
+import React from 'react';
+import { storiesOf } from '@storybook/react';
+import { action } from '@storybook/addon-actions';
+import InfiniteDirectoryList from './';
+
+const items = [
+  { id: '1', type: 'Folder', date: 'Mar 29, 2019 3:34 PM', name: 'A folder' },
```
Comment (atb91590) [inline packages/veritone-react-common/src/components/InfiniteDirectoryList/story.js]: update your story to conform to the correct format of items in the spec

File: packages/veritone-react-common/src/components/InfiniteWrapper/styles.scss [hunk 3]
```diff
@@ -0,0 +1,4 @@
+.infinte-wrapper {
+  height: 300px;
```
Comment (atb91590) [inline packages/veritone-react-common/src/components/InfiniteWrapper/styles.scss]: Do not use a static height. Height should be inherited from its parent and still work with the scroll loader

File: packages/veritone-react-common/src/components/InfiniteDirectoryList/styles.scss [hunk 4]
```diff
@@ -0,0 +1,20 @@
+.loading-container {
```
Comment (atb91590) [inline packages/veritone-react-common/src/components/InfiniteDirectoryList/styles.scss]: Use camel case class names. This allows easier access to styles (styles.loadingContainer).

`loading-container` => `loadingContainer`

Comment (seta-tuha) [reply packages/veritone-react-common/src/components/InfiniteDirectoryList/styles.scss]: Actually, scss convention className is not using camelCase
http://bdavidxyz.com/blog/how-to-name-css-classes/

**Comment by seta-tuha** [comment]
Merged into Data Picker

**Pull request closed without merging**